### PR TITLE
fix(DB/creature) Add Magnetic Pull to Living Cyclone and remove wrong…

### PR DESCRIPTION
**Pull request by v-mstrs**
… spell from Bladespire Battlemage

## Changes Proposed:
-  Add Magnetic Pull to Living Cyclone, adjusts Lightning Shield
-  Removes wrong spell (Lightning Shield) from Bladespire Battlemage

## Issues Addressed:

- Closes https://github.com/chromiecraft/chromiecraft/issues/5623
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/16282

## SOURCE:

https://www.wowhead.com/wotlk/npc=17160/living-cyclone
https://www.wowhead.com/wotlk/npc=19996/bladespire-battlemage
## Tests Performed:

- Works

## How to Test the Changes:

1. .npc add 17160, aggro

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by Gultask** [comment]
Is it supposed to not repeat?
Mangos has it cast ON_RANGE 8-40 with a repeat timer of 10000-15000. ON_RANGE also casts it immediately on first cast.

**Comment by v-mstrs** [comment]
Maybe I did something wrong but it casts only once and the Lightning Shield refreshes one in combat immediately then out of combat.

**Comment by Gultask** [comment]
@v-mstrs I think you placed the file in the wrong folder. It isn't being detected. It needs to be in the pending folder

File: rev_1684961632752120124.sql [hunk 1]
```diff
@@ -0,0 +1,7 @@
+--
+DELETE FROM `smart_scripts` WHERE (`source_type` = 0 AND `entryorguid` = 17160);
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(17160, 0, 0, 0, 23, 0, 100, 1, 12550, 0, 10000, 10000, 0, 11, 12550, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Living Cyclone - On Missing Aura - Cast Lightning Shield'),
```
Comment (Gultask) [inline rev_1684961632752120124.sql:4]: It's only happening once because it has event_flag 1 (No Repeat), causing it to not repeat in the same fight

Comment (Gultask) [reply rev_1684961632752120124.sql:4]: Same for the Magnetic Pull

**Review by Gultask — APPROVED** [review]

**Pull request merged**